app/controllers/social_networking/shared_item_controller.rb

```diff
@@ -1,6 +1,8 @@
 module SocialNetworking
   # Manage shared items
   class SharedItemController < ApplicationController
+    SHARED_ITEM_PAGE_SIZE = 5
+
     skip_before_action :verify_authenticity_token
     def hide
       shared_item_to_hide = SharedItem.find_by_id(shared_item_params[:id])
```

Comment:
The primary difference is that `find` will throw `ActiveRecord::RecordNotFound` when that id doesn't exist, and `find_by_id` will return `nil`.